Gems/Atom/Feature/Common/Assets/ShaderLib/Atom/Features/PBR/Lights/PointLight.azsli

```diff
@@ -103,7 +103,7 @@ void ApplyPointLight(ViewSrg::PointLight light, Surface surface, inout LightingD
 
 #if ENABLE_TRANSMISSION
             // o_transmission_mode == NONE is not taken into account because GetBackLighting already ignores this case
-            if (o_transmission_mode == TransmissionMode::ThickObject)
+            if (o_transmission_mode == TransmissionMode::ThickObject || o_transmission_mode == TransmissionMode::Burley)
```

Comment:
This is a mistake, right? The || TransmissionMode::Burley should go with the TransmissionMode::ThinObject, not TransmissionMode::ThickObject like you have here, correct?

Reply:
+1

Reply:
@antonmic @adiblev Yes, it is a mistake, now fixed.